### ISSUE TITLE
[Connect] Styling issues

### ISSUE THREAD

**Issue by jasperjansz** [leather-io/extension#92]
- [x] Text box 2 lines instead of 3 (68px instead of 80px)
- [x] Hide text box drag thing on the bottom right
- [x] "Didn't save your SK?.." centered 24px from the bottom
- [x] "Didn't save your SK?.." font size 14px (body regular & medium for link)
- [x] Button font-size 14px

It would be nice to have this title in such a way that we avoid a lone word on the second line in most cases. I did this by giving it extra padding (72px to the edge of the popup).

**Comment by aulneau** [comment]
This issue is a bit hard to parse, could you include screenshots? Additionally, these [connect] tickets should probably go into that repo -> https://github.com/blockstack/connect/issues

**Comment by aulneau** [comment]
> "Didn't save your SK?.." centered 24px from the bottom

This is inconsistent between mobile and desktop. Can we opt to keep them the same (mobile treatment, 24 px below input)
